src/main/flight/pid.c

```diff
@@ -1355,9 +1360,9 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
         if (maxVelocity[axis]) {
             currentPidSetpoint = accelerationLimit(axis, currentPidSetpoint);
         }
-        // Yaw control is GYRO based, direct sticks control is applied to rate PID
+        // Yaw control is GYRO based, direct sticks control is applied to rate PID (for NFE Race Mode pitch is also gyro based)
```

Comment:
Rather than referring to "NFE Race Mode" (which has no context here), it would be better to describe the function in terms of what it does.

Reply:
Changed the comment